modules/integration/src/test/scala/scala/cli/integration/CompileTestDefinitions.scala

```diff
@@ -720,4 +720,20 @@ abstract class CompileTestDefinitions
       expect(buildTargetDirs.size == 2)
     }
   }
+
+  if (!Properties.isWin)
+    // TODO: make this work on Windows
```

Comment:
- likely the same problem as https://github.com/VirtusLab/scala-cli/issues/2973, let's track it there

Reply:
sure, added the ticket link to the comment